pkg/batchmapper/examples/batchmap_flatmap/go.mod

```diff
@@ -1,6 +1,6 @@
 module batchmap-flatmap
 
-go 1.22
+go 1.20
```

Comment:
was 1.22 not working?

Reply:
im assuming it would, I just changed it to follow the version of all the other examples + im running go 1.20 so I wouldn't be able to build it without upgrading my go version

Reply:
since the other examples are also go 1.20 im assuming this would be the case for most numaflow developers anyways

Reply:
sure sure, was just curious. It's fine.